include/akushon/action/model/action_name.hpp

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2021 Ichiro ITS
+// Copyright (c) 2023 Ichiro ITS
```

Comment:
It should be updated (`2021-2023`) instead of replaced (just `2023`).

And it should match with the declaration in [the license file](https://github.com/ichiro-its/akushon/blob/master/LICENSE).